sdk/core/Azure.Core/tests/FailedResponseExceptionTests.cs

```diff
@@ -336,21 +342,24 @@ public void ParsesJsonErrors_ResponseCtor()
         }
 
         [Test]
-        public void ParsesJsonErrors_ResponseCtor_Stream([Values(true, false)] bool canSeek)
+        public void ParsesJsonErrors_ResponseCtor_Stream([Values(true, false)] bool canSeek, [Values(true, false)] bool hasErrorWrapper)
         {
             var formattedResponse =
                 "Custom message" + s_nl +
                 "Status: 210 (Reason)" + s_nl +
                 "ErrorCode: StatusCode" + s_nl +
                 s_nl +
                 "Content:" + s_nl +
-                "{ \"error\": { \"code\":\"StatusCode\", \"message\":\"Custom message\" }}" + s_nl +
+            (hasErrorWrapper ? "{ \"error\": { \"code\":\"StatusCode\", \"message\":\"Custom message\" }}" :
```

Comment:
Is it worth adding a test that `Target` etc get populated?  I think it will work now.

Reply:
Is there test coverage for it elsewhere?  If so, then I agree it doesn't need to happen here.

Reply:
We do have coverage that non-standard json gets incorporated into the formatted message. Is there some other coverage are you thinking of in particular? In other words, what about RFE's behavior changes that is unique to when `Target` is populated?